Sources/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.7)
+cmake_minimum_required(VERSION 3.7.2)
```

Comment:
Are you sure this is the minimum required version?

Reply:
Technically 2.8.7 is still the minimum required version, but I made this change to get rid of this annoying warnining: 
```
CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of CMake.
  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.
```

Reply:
I chose 3.7.2 because it is the version from debian oldoldstable so I don't think anyone is still using something older. But the version can be changed.

Reply:
Ok, fine for me.